lib/widgets/custom_drawer.dart

```diff
@@ -66,7 +66,7 @@ class CustomDrawer extends StatelessWidget {
                             ),
                           ),
                           SizedBox(
-                            height: SizeConfig.screenHeight! * 0.45,
+                            height: SizeConfig.screenHeight! * 0.41,
```

Comment:
have you tried this size tweaking with the smallest device size possible?

Reply:
Yes, moreover I think  the SizeConfig will handle it for variable screen sizes.

Reply:
Ya size config takes care of it but for it to work efficiently things need to be checked on the smallest device size possible.
For eg. listtile height is not defined explicitly and flutter has a static size for it irrespective of the device screen size. hence in this case we have a contain header on top with variable height, a listview with variable height but the listtiles used are static height. hence they may cause problems if not tested with the smallest device size.
Rest changes are GTG.

Reply:
Yes, I understand some components are static and have predefined dimensions in flutter. Let me run it on a smallest device available in my avd and report you here,

Reply:
Thanku @CyberWake !  I didn't keep it in mind! Now new changes are done and converted it into list view which will be active for small devices. Now it works on any small device (or the smallest available in my AVD ;) ). 
Let me know what shall I do for the new code merging. (Shall I make a new pull request or there is other option). Now the code works great.